Updating how to pass env variables (#4945)
Kong/docs.konghq.com: app/_src/gateway/kong-enterprise/secrets-management/backends/env.md

{vault://env/pg-creds/password}
```

{:.note}

> > While adding the environment variable inside using helm, make sure the varible which is been passed has kong- appended to it as this is been appended to all the environment variable by default.

Reviewer: ```suggestion
> When adding an environment variable with Helm, ensure that the variable being passed has `kong-` appended to it. 
```
